lib/environment_file.rb

```diff
@@ -4,6 +4,7 @@ class EnvironmentFile
     network_watcher
     msi
     sql
+    mysql
```

Comment:
This is fine to bundle with the `sql` option. You can update your controls to `only_if { ENV['SQL'] }`